### PR TITLE
feat(website): add typekit demo page

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Implement a demo page for the Typekit

Head over to the new page in the demo to see it in action
image

### Potential Breaking Changes

None

### CODE REVIEW

File: packages/website/src/pages/index.tsx [hunk 1]
```diff
@@ -57,9 +57,9 @@ const FoundationsPages: React.FC = () => (
                 thumbnail="iconography"
             />
             <Tile
-                title="Headings"
-                description="A heading is a title at the top of a page or section. Its distinctive font helps visualize the hierarchy of information."
-                href="foundations/Headings"
+                title="Typekit"
+                description="The Typekit covers the entire spectrum of typography styles designed specifically to work in harmony with the Plasma ecosystem."
```
Comment (gdostie) [inline packages/website/src/pages/index.tsx:61]: @jlevesquecoveo I improvised something with the description here, feel free to change it up 😄

**Review by aiheon — APPROVED** [review]
Shoud we add the github link to the scss file? :eyes:

**Comment by gdostie** [comment]
> Shoud we add the github link to the scss file? 👀

Oh yes good idea I will add it 👍

File: packages/website/src/building-blocs/PageHeader.tsx [hunk 2]
```diff
@@ -14,6 +14,12 @@ export interface PageHeaderProps {
      * @example '/button/Button.tsx'
      */
     componentSourcePath?: string;
+    /**
+     * Path to a relevant source file in the repo
```
Comment (aiheon) [inline packages/website/src/building-blocs/PageHeader.tsx:18]: :tada:

**Review by jlevesquecoveo — APPROVED** [review]

**Review by GermainBergeron — APPROVED** [review]

**Pull request merged**